priv/repo/seeds.exs

```diff
@@ -9,3 +9,384 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+alias Lunchbot.LunchbotData
+alias Lunchbot.LunchbotData.Menu
```

Comment:
This could also be written as:
```
alias Lunchbot.LunchbotData.{
  Category,
  Menu,
  MenuCategories,
  Item,
  OptionHeadings,
  ItemOptionHeadings,
  Options
}
```
which would save multiple calls to the alias function.